boards/weact-f401cc/doc.txt

```diff
@@ -33,6 +33,19 @@ It is available on sites like AliExpress for less than 3€.
 | Datasheet        | [Datasheet](https://www.st.com/resource/en/datasheet/stm32f401cc.pdf) |
 | Reference Manual | [Reference Manual](https://www.st.com/content/ccc/resource/technical/document/reference_manual/9b/53/39/1c/f7/01/4a/79/DM00119316.pdf/files/DM00119316.pdf/jcr:content/translations/en.DM00119316.pdf) |
 
+## Implementation Status
+
+| Device | ID        | Supported | Comments  |
+|:------------- |:------------- |:------------- |:------------- |
+| MCU        | STM32F401CC   | partly    | Energy saving modes not fully utilized |
+| Low-level driver | GPIO    | yes       | |
+|        | PWM       | yes (4 pins available))   |  |
+|        | UART      | 2 UARTs       | USART2 via  PA3(RX)/PA2(TX), USART1 on PA10(RX)/PA9(TX) |
+|        | I2C       | 1 I2C     | |
+|        | SPI       | 1 SPI     | |
+|        | USB       | yes        | |
+|        | Timer     | 1 32 bit timer (TIM5)     | |
```

Comment:
```suggestion
| Device            | ID            | Supported                 | Comments                                                  |
|:----------------- |:------------- |:------------------------- |:--------------------------------------------------------- |
| MCU               | STM32F401CC   | partly                    | Energy saving modes not fully utilized                    |
| Low-level driver  | GPIO          | yes                       |                                                           |
|                   | PWM           | yes (4 pins available))   |                                                           |
|                   | UART          | 2 UARTs                   | USART2 via  PA3(RX)/PA2(TX), USART1 on PA10(RX)/PA9(TX)   |
|                   | I2C           | 1 I2C                     |                                                           |
|                   | SPI           | 1 SPI                     |                                                           |
|                   | USB           | yes                       |                                                           |
|                   | Timer         | 1 32 bit timer (TIM5)     |                                                           |
```

IMO its easier to read and update the source of the table if the columns are aligned.